src/main/main.py

```diff
@@ -0,0 +1,111 @@
+from src.main import consts, activity_tracker_handler as ath, data_handler as dh
+import pandas as pd
+import logging
+import csv
+import sys
+import os
+
+pd.set_option('display.max_rows', 250)
+pd.set_option('display.max_columns', 100)
+
+
+def __get_data_path():
+    args = sys.argv
+    if len(args) < 3 or consts.PATH_CMD_ARG not in args:
+        raise NameError('List of arguments must have required 1 element: path!')
+    try:
+        path = args[args.index(consts.PATH_CMD_ARG) + 1]
+    except ValueError:
+        raise NameError('List of arguments does not have the argument -path')
+    except IndexError:
+        raise NameError('List of arguments does not have a value after the argument -path')
+    return path
+
+
+def __get_real_ati_file_index(path: str, files: list, ati_key: str):
+    sniffer = csv.Sniffer()
+    sample_bytes = 1024
+    count_ati = 0
+    ati_index = -1
+    for i, f in enumerate(files):
+        if ati_key in f:
+            if not sniffer.has_header(open(path + '/' + f, encoding=consts.ENCODING).read(sample_bytes)):
+                count_ati += 1
+                ati_index = i
+                if count_ati >= 2:
+                    raise ValueError('Count of activity tracker files is more 1')
+    return ati_index
+
+
+def __separate_ati_and_other_files(files: list, folder: str, full_path: str):
+    ati_file_index = __get_real_ati_file_index(full_path, files, consts.ACTIVITY_TRACKER_FILE_NAME)
+    ati_file = None
+    ati_id = None
+    if ati_file_index != -1:
+        ati_file = files[ati_file_index]
+        del files[ati_file_index]
+        ati_id = folder.split('_')[1]
+    return files, ati_file, ati_id
+
+
+def __get_folders_for_handling(path: str):
+    # Get child folders for the root folder from generator and remove result folder
+    return list(filter(lambda x: consts.RESULT_FOLDER not in x, next(os.walk(path))[1]))
+
+
+def __write_result(path: str, file: str, result_df: pd.DataFrame):
```

Comment:
I think no because we don't have the activity tracker files when we have finished handling it and it is strange to save a folder for it or not? But of course, I can save the structure if it is necessary. Is it important to put the code tracker files from one user in the one folder?